docs/docsite/rst/intro_windows.rst

```diff
@@ -178,6 +178,15 @@ To see what tickets if any you have acquired, use the command klist
 
    klist
 
+Automatic kerberos ticket management
+------------------------------------
+
+Ansible defaults to automatically managing kerberos tickets (as of Ansible 2.3) when both username and password are specified for a host that's configured for kerberos. A new ticket is created in a temporary credential cache for each host, before each task executes (to minimize the chance of ticket expiration). The temporary credential caches are deleted after each task, and will not interfere with the default credential cache.
```

Comment:
I actually wrote it that way originally, but edited it to this later. I'd rather lead with the details, not the minutiae- the usefulness of that clause will decline swiftly over the next few months to a point where it'll be nearly pointless anyway (especially with versioned docs- if we had that now, I probably would've excluded it entirely).